Fundamentals 1/data-types.html/data-types.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Data Types</title>
+    <style>
+        h1, h3 {
+            text-align: center;
+        }    
+         h3 {
+             margin-top: 100px;
+         }    
+        table{
+            font-size: 20px;
+            border-collapse: collapse;
+            width: 50%;
+            margin: 0 auto;
+        }
+        th {
+            border: 1px solid #dddddd;
```

Comment:
You can always trim a few characters with hex colors that have the same value:

#dddddd -> #dddd
#ffffff -> #ffff

and so on...

[Reference](https://developer.mozilla.org/en-US/docs/Web/CSS/color#syntax)